services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
The type hint 'list[int]' is used here, which is correct for Python 3.9 and above. Ensure that your environment supports this syntax. If not, consider using 'List[int]' from the 'typing' module.